packages/shared_preferences/shared_preferences_linux/CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
+## 0.0.2
```

Comment:
Nits: I think this should be just a patch? 0.0.1+1

Reply:
I want to be able to specify the dependency in the endorsement so that it requires this version; I assumed build numbers weren't valid for that purpose so did a minor version instead.